### DISCUSSION TITLE
Output Radioss file when running Openradioss with LS-DYNA files

### DISCUSSION THREAD

**Discussion by mhm39** [OpenRadioss/OpenRadioss#587] [Q&A] [answered]
Hi,
fantastic news that OpenRadioss is now available, thanks! I am trying to run my LS-DYNA files, and obviously some things do not translate that easily. As it may well possible to develop some workarounds directly in Radioss format, I was wondering whether there is some way of creating translated Radioss files from the LS-DYNA input files.

-mhm39

**Comment by elequiniou** [comment, marked as answer]
Hi, first, thanks a lot for your kind words about OpenRadioss! Sure, there are some ways to generate Radioss input from LS-DYNA one using Altair HyperWorks preprocessor which includes such converter.  However, with the support of this direct reading by OPenRadioss, the idea is more to allow user to stay with this alternative input format. Looking at the list of current supported keywords, you may replace some unsupported option by another comparable one which is already supported. Also, please, feel free to share which cards you have issue with. The list of supported keywords is continuously increased and with the help of the community we aim to speedup such support!

**Reply by mhm39** [reply]
Hi, thanks for the quick reply. In fact my question was triggered by the fact that OpenRadioss sometimes does not consider all possible input fields of a given LS-Dyna keyword. For example with *CONTROL_TIMESTEP only tssfac and dt2ms is considered. Some other fields can easily be approximated by using native Radioss keywords, this however requires editing the ...0001.rad file between the starter and the engine runs. So it would be simpler if one could include native Radioss somewhere. Generally speaking, if there are some problems reading the LS-Dyna input, it would be helpful to see what OpenRadioss makes of it, this would greatly simplify the task of adapting the original model.
I attach test.k to illustrate
best regards and thanks
-m
[test.k.txt](https://github.com/OpenRadioss/OpenRadioss/files/9753763/test.k.txt)

**Reply by elequiniou** [reply]
Thanks for your feedback. You are right, regarding the 1.rad automatically generated, as you noticed and proposed the workaround is to add by hand the additional Radioss comparable options directly in the 1.rad before running the engine. So, this becomes somehow an hybrid input. This is something we are considering to implement in a more generic way, to allow to mix input formats.
Best Regards,
Eric.

**Reply by jparraporcar** [reply]
Hello!
Thanks for the discussion. Just to follow-up on this topic. Is it already possible to create a mix input of lsdyna/radioss format to launch in OpenRadioss? and... is there a way to obtain the mapped .rad file once the mapping has been done to see the result of the mapping as if the .rad file would have been created manually?
Thanks in advance!
Jordi

**Reply by SyazwanSamad** [reply]
@jparraporcar I have tried the mixing of LS Dyna into OpenRadioss using /INCLUDE_LS-DYNA and vice versa using *INCLUDE_RADIOSS. Both works well.

**Reply by jparraporcar** [reply]
@SyazwanSamad thanks for the response, I will try it out!

**Reply by avocadofruit** [reply]
@SyazwanSamad  Can I check, how did you include the "/INCLUDE_LS-DYNA" command into your Radioss file? Does it just come before the Radioss command?

**Reply by SyazwanSamad** [reply]
Hi @avocadofruit,
It should be placed inside the *0000.rad file. Example here i just placed it before /END keyword.
![image](https://github.com/user-attachments/assets/c4d3ea85-aa6a-46b2-a240-ac6a28e94c18)

**Reply by okkarkyaw122** [reply]
Hello. I have one issue. I run a simple 3 point bending test which includes rigid body motion and MAT_ADD_Erosion. The file run smoothly in LS dyna solver and shows the expected result. However after running in Radioss, the analysis is completed but and there are output vtk files as well. However,there is no movement or no results at all. May I know the reason why?

**Reply by laurent-altr** [reply]
Hello @okkarkyaw122, 
Could you create a new issue, and provide the model?

Thank you, 
Laurent

**Reply by okkarkyaw122** [reply]
Thank you @laurent-altr. I have created a new session and provided the model. 
here is the link to the discussion https://github.com/orgs/OpenRadioss/discussions/3115.
